<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="wiseoper" duration-ms="21885" started-at="2020-09-10T02:37:56Z" finished-at="2020-09-10T02:38:18Z">
    <groups>
    </groups>
    <test name="applyContinueTest" duration-ms="21885" started-at="2020-09-10T02:37:56Z" finished-at="2020-09-10T02:38:18Z">
      <class name="test.ApplyContinueTest">
        <test-method status="PASS" signature="setUp(java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:test.ApplyContinueTest@5f282abb]" name="setUp" is-config="true" duration-ms="7347" started-at="2020-09-10T10:37:56Z" finished-at="2020-09-10T10:38:03Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[http://www.soomey.net/web/sq/#/passport/login]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="testApplyContinue()[pri:0, instance:test.ApplyContinueTest@5f282abb]" name="testApplyContinue" duration-ms="14203" started-at="2020-09-10T10:38:03Z" finished-at="2020-09-10T10:38:18Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[检查点不通过 expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 检查点不通过 expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertTrue(Assert.java:41)
	at test.ApplyContinueTest.testApplyContinue(ApplyContinueTest.java:36)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:414)
	at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testApplyContinue -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:test.ApplyContinueTest@5f282abb]" name="tearDown" is-config="true" duration-ms="200" started-at="2020-09-10T10:38:18Z" finished-at="2020-09-10T10:38:18Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- test.ApplyContinueTest -->
    </test> <!-- applyContinueTest -->
  </suite> <!-- wiseoper -->
</testng-results>
